artemis/src/main/java/tech/pegasys/artemis/GenesisCommand.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis;
+
+import static tech.pegasys.artemis.datastructures.util.SimpleOffsetSerializer.serialize;
+import static tech.pegasys.artemis.util.alogger.ALogger.STDOUT;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+import org.apache.logging.log4j.Level;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.datastructures.util.MockStartValidatorKeyPairFactory;
+import tech.pegasys.artemis.statetransition.util.StartupUtil;
+import tech.pegasys.artemis.util.bls.BLSKeyPair;
+import tech.pegasys.artemis.util.cli.VersionProvider;
+
+@Command(
+    name = "genesis",
+    description = "Commands for generating genesis state",
+    abbreviateSynopsis = true,
+    mixinStandardHelpOptions = true,
+    versionProvider = VersionProvider.class,
+    synopsisHeading = "%n",
+    descriptionHeading = "%nDescription:%n%n",
+    optionListHeading = "%nOptions:%n",
+    footerHeading = "%n",
+    footer = "Artemis is licensed under the Apache License 2.0")
+public class GenesisCommand {
+
+  @Command(
+      name = "mock",
+      description = "Generate a mock genesis state",
+      mixinStandardHelpOptions = true,
+      abbreviateSynopsis = true,
+      versionProvider = VersionProvider.class,
+      synopsisHeading = "%n",
+      descriptionHeading = "%nDescription:%n%n",
+      optionListHeading = "%nOptions:%n",
+      footerHeading = "%n",
+      footer = "Artemis is licensed under the Apache License 2.0")
+  public void generate(@Mixin MockGenesisParams params) throws IOException {
+    FileOutputStream fileStream = new FileOutputStream(params.outputFile);
```

Comment:
Probably best to use a try-with-resources here (`try (FileOutputStream fileStream = new FileOutputStream(params.outputFile) {`) to ensure the stream is automatically closed even if exceptions are thrown.  Probably doesn't really matter in this case as the JVM will exit once this method ends anyway but good to follow the usual pattern.